packages/origin-247-certificate/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@energyweb/origin-247-certificate",
-    "version": "4.1.0",
+    "version": "4.1.1",
```

Comment:
I think it should be bumped as we release new version after merging in the changes

Reply:
done in commit [409c3fe](https://github.com/energywebfoundation/origin-247-sdk/pull/104/commits/409c3fe8924a68d81b7a84780afc12f217deedf1)